Model/Config/Source/YotpoOrderStatusBuilder.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Yotpo\Yotpo\Model\Config\Source;
+
+use Magento\Framework\Option\ArrayInterface;
+
+class YotpoOrderStatusBuilder implements ArrayInterface
+{
+    protected $_manager;
+
+    public function __construct(\Magento\Sales\Model\ResourceModel\Order\Status\Collection $manager)
+    {
+        $this->_manager = $manager;
+    }
```

Comment:
why is this class necessary? why can't we use the \Magento\Sales\Model\ResourceModel\Order\Status\Collection
without the wrapping